spec/support/usps_ipp_helper.rb

```diff
@@ -180,10 +180,15 @@ def request_unexpected_expired_proofing_results_args
     }
   end
 
-  def stub_request_unexpected_invalid_applicant
-    stub_request(:post, %r{/ivs-ippaas-api/IPPRest/resources/rest/getProofingResults}).to_return(
-      **request_unexpected_invalid_applicant_args,
+  def stub_request_unexpected_invalid_applicant(response_body_overrides = {})
+    response = merge_into_response_body(
+      request_unexpected_invalid_enrollment_code_args,
+      response_body_overrides,
     )
+    stub_request(
+      :post,
+      %r{/ivs-ippaas-api/IPPRest/resources/rest/getProofingResults},
+    ).to_return(response)
   end
 
   def request_unexpected_invalid_applicant_args
```

Comment:
I might be wrong but it looks like this is no longer being used. You have a similar but different stub. Wonder if method `stub_request_unexpected_invalid_applicant` should still use it or fine calling your new method and we can delete this?

Reply:
I actually messed this up... it was a copy pasta... I will fix it.

Reply:
I wondered if it was a copy pasta job. 🍝   Thanks for the fix. I see it now being used in a spec with context that is fitting. Resolving comment now.